Code/GraphMol/FMCS/Wrap/rdFMCS.cpp

```diff
@@ -36,17 +40,31 @@ struct PyMCSAtomCompare {
                                  const ROMol& mol2, unsigned int atom2) const {
     return RDKit::checkAtomChirality(p, mol1, atom1, mol2, atom2);
   }
+  virtual bool operator()(const MCSAtomCompareParameters&,
+                          const ROMol&, unsigned int, const ROMol&, unsigned int) {
+    PyErr_SetString(PyExc_AttributeError,
+      "The " COMPARE_FUNC_NAME "() method "
+      "must be overridden in the rdFMCS.MCSAtomCompare subclass");
+    python::throw_error_already_set();
+    return false;
+  }
+  // DEPRECATED: remove from here in release 2021.01
```

Comment:
This is a nit, but it confused me for a bit while reviewing this: 
The comment that accompanies this one (on line 60) really looks like it applies to the method below it.  I think it would be clearer if you remove that comment and change this one to:
```suggestion
  // DEPRECATED: remove this method in release 2021.01
```